Doc/library/io.rst

```diff
@@ -503,6 +515,40 @@ I/O Base Classes
       If the object is in non-blocking mode and no bytes
       are available, ``None`` is returned.
 
+   .. method:: backread(size=-1, /)
+
+      Read up to *size* bytes from the *end* of the object and return them.
+      As a convenience, if *size* is unspecified or -1, all bytes until BOF
```

Comment:
I had never heard of "BOF", maybe make this more explicit?

```suggestion
      As a convenience, if *size* is unspecified or -1, all bytes until start
```

or just leave it out?

```suggestion
      As a convenience, if *size* is unspecified or -1, all bytes
```

Reply:
BOF is like EOF (**B**eginning of the file) but it rarer and is mainly used in VBA (more precisely in database records). I used this because I thought it was common enough and intuitive enough (BOF/EOF) but it appears not. I'll update all the docs at once in the next commit.

Reply:
Ah I only updated the Python and C implementation for now (I fixed a ref warning in the meantime)